hydroflow/src/scheduled/graph_ext.rs

```diff
@@ -18,6 +21,10 @@ macro_rules! subgraph_ext {
         ( $($recv_param:ident : $recv_generic:ident),* ),
         ( $($send_param:ident : $send_generic:ident),* )
     ) => {
+        /// Adds a subgraph with specific topology:
+        ///
```

Comment:
intentional?

Reply:
Yeah it connects with the `#[doc = ...]` line